src/languages/es.js

```diff
@@ -827,6 +827,9 @@ export default {
         privateDescription: 'Sólo las personas que están invitadas a esta sala pueden encontrarla',
         createRoom: 'Crea una sala de chat',
         roomAlreadyExists: 'Ya existe una sala con este nombre',
+        pleaseEnterRoomName: '',
+        pleaseSelectWorkspace: '',
+        pleaseSelectVisibility: '',
```

Comment:
Need them...

Reply:
Can we ask in `#expensify-open-source` for correct translations? I'm not sure what the process is but someone should be able to help.

Reply:
@ABee-Tech Asked them, waiting for reply...

Reply:
I have also asked on the channel.